code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -306,6 +306,20 @@
 					msg += "[t_He] [t_is] flushed and wheezing.\n"
 				if (bodytemperature < dna.species.bodytemp_cold_damage_limit)
 					msg += "[t_He] [t_is] shivering.\n"
+				if (HAS_TRAIT(src, TRAIT_PACIFISM))
+					msg += "[t_He] doesn't look like [t_He] would hurt a fly.\n"
+				if (HAS_TRAIT(src, TRAIT_CLUMSY))
+					msg += "[t_He] looks like a clutz.\n"
+				if (HAS_TRAIT(src, TRAIT_GRABWEAKNESS))
+					msg += "[t_He] seems like a pushover.\n"
+				if (HAS_TRAIT(src, TRAIT_SKITTISH))
+					msg += "[t_He] is a bit skittish.\n"
+				if (HAS_TRAIT(src, TRAIT_ANXIOUS))
+					msg += "[t_He] looks awkward.\n"
```

Comment:
"awkward" seems a bit, well, awkwardly used here. would "anxious" work better?

Reply:
how does "nervous" sound was thinking about that instead originally.

Reply:
Something that leans into the "social" part of it being social anxiety perhaps? Like "seems uncomfortable around people" or even "has trouble making eye contact"?

Reply:
I do like that but feels long, does "seems uncomfortable" sound right?

Reply:
genders are handled automatically [t_He] is she, he, it depending on their gender so best to imagine any "theys" as that.

Reply:
I forgot to consider all pronouns in original wording, reworded all descriptions and changed "awkward" to "nervous" along with a few other things, will mark this as resolved for now.